documentation/docs/core-concepts/hello-world.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Hello World
+path: /hello-world/
+---
+
+# Hello World
+
+A simple React app using Fusion.js would look something like this:
+
+```js
+import App from 'fusion-react';
+import React from 'react';
+
+export default async function start() {
+  const root = <div>Hello World</div>
+  const app = new App(root);
+  return app;
+}
+```
+
+Running this code in your browser would produce the string "Hello World" on a plain white background with no styling.
```

Comment:
Would recommend something like:

"Running this code (`fusion dev`) will produce the string..."

Reply:
👍